---
layout: default
title: Wade R. Roberts
---
{% include section.html %}

I am an evolutionary biologist, working in the [Roalson Lab](http://roalsonlab.weebly.com) at [Washington State University](http://www.wsu.edu) in the [School of Biological Sciences](http://www.sbs.wsu.edu) and the [Molecular Plant Sciences Graduate Program](http://www.mps.wsu.edu). I combine bioinformatics, genomics, and phylogenetics to understand patterns of biological diversification and the evolutionary and ecological processes that generate them.

{% include button.html label="CV" %}

{% include section.html header="Research" %}

Understanding diversification at the macro- and microevolutionary scales has long been a central goal of evolutionary biology. My research aims to understand the patterns and processes of diversfication. I use comparative genomics and phylogenomics to investigate how underlying genetic networks have evolved under natural selection to produce diverse phenotypes. I am particularly interested in the evolution of floral form in the genus _Achimenes_ (Gesneriaceae), a colorful and diverse genus native to Mexico and Central America.

{% include button.html label="Details" link="/research" %}

{% include section.html header="Publications" %}

**Roberts W.R.**, Roalson E.H. 2018. Phylogenomic analyses reveal extensive gene flow within the magic flowers (_Achimenes_, Gesneriaceae). American Journal of Botany in Press.

**Roberts W.R.**, Roalson E.H. 2017. Comparative transcriptome analyses of flower development in four species of _Achimenes_ (Gesneriaceae). BMC Genomics 18: 240 [[doi]](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3623-8)

Roalson E.H., **Roberts W.R.** 2016. Distinct Processes Drive Diversification in Different Clades of Gesneriaceae. Systematic Biology 64: 662-684. [[doi]](https://academic.oup.com/sysbio/article/65/4/662/1753444) [[cover image]](http://sysbio.oxfordjournals.org/content/65/4.cover-expansion)

{% include button.html label="On Google Scholar" link=site.author.scholar icon="fa-graduation-cap" %}

{% include section.html header="Projects" %}

### [TACOCAT](https://github.com/wrroberts/TACOCAT)
**T**rim **A**nd **CO**nstruct **C**ombined **A**ssembly **T**ranscriptome

TACOCAT was built to streamline the de novo transcriptome assembly process and create a robust reference for downstream analysis. This pipeline starts with raw paired-end Illumina reads in fastq format, quality trims the reads, assembles the reads using multiple assemblers and different k-mer sizes, and outputs a reference classified into primary and alternate sets of transcripts.

### [Rosalind](https://github.com/wrroberts/rosalind)

My worked solutions to ROSALIND bioinformatics problems.

{% include button.html label="Fork Me" link=site.author.github icon="fa-github" %}
