---
layout: default
title: Wade R. Roberts
---
{% include section.html %}

I am an evolutionary biologist, working in the [Roalson Lab](http://roalsonlab.weebly.com) at [Washington State University](http://www.wsu.edu) in the [School of Biological Sciences](http://www.sbs.wsu.edu) and the [Molecular Plant Sciences Graduate Program](http://www.mps.wsu.edu). I combine bioinformatics, computational biology, genomics, and phylogenetics to understand patterns of biological diversification and the evolutionary and ecological processes that generate them.

{% include section.html header="Research" %}

My work aims to understand the evolution of floral diversity, particularly in the African violet family Gesneriaceae. Understanding the genetic mechanisms that produce diversity is one of the central goals of evolutionary biology. Flowers, with their vast array of shapes and colors, represent one of the most amazing and visible products of evolution. Floral biology can be studied through both an adaptive perspective, particulary as a function of reproductive biology and pollination, or a genetic perspective, particularly molecular analyses of developmental and biochemical pathways. I aim to understand the diversification of floral form using comparative genomics, phylogenetics, and developmental genetics.

{% include button.html label="Details" link="/research" %}

{% include section.html header="Publications" %}

**Roberts W.R.**, Roalson E.H. 2017. Comparative transcriptome analyses of flower development in four species of _Achimenes_ (Gesneriaceae). BMC Genomics. Accepted.

Roalson E.H., **Roberts W.R.** 2016. Distinct Processes Drive Diversification in Different Clades of Gesneriaceae. Systematic Biology 64 (40): 662-684. [[doi]](http://sysbio.oxfordjournals.org/content/early/2016/02/14/sysbio.syw012.abstract) [[cover image]](http://sysbio.oxfordjournals.org/content/65/4.cover-expansion)

{% include button.html label="On Google Scholar" link=site.author.scholar icon="fa-graduation-cap" %}

{% include section.html header="Projects" %}

### [TACOCAT](https://github.com/wrroberts/TACOCAT)
**T**rim **A**nd **CO**nstruct **C**ombined **A**ssembly **T**ranscriptome

TACOCAT was built to streamline the de novo transcriptome assembly process and create a robust reference for downstream analysis. This pipeline starts with raw paired-end Illumina reads in fastq format, quality trims the reads, assembles the reads using multiple assemblers and different k-mer sizes, and outputs a reference classified into primary and alternate sets of transcripts.

### [Rosalind](https://github.com/wrroberts/rosalind)

My worked solutions to ROSALIND bioinformatics problems.

{% include button.html label="Fork Me" link=site.author.github icon="fa-github" %}
